code/modules/research/protolathe.dm

```diff
@@ -121,21 +121,20 @@
 	if(default_part_replacement(I, user))
 		return
 	if(I.is_open_container())
-		return 1
```

Comment:
This is the only true change in this PR - if proc returns 1 in this case (e.g. when you trying to click on protolathe with beaker) nothing will happen. 0 allow reagents transfering.